scripts/check-ci

```diff
@@ -13,10 +13,15 @@ exit_() {
     exit 1
 }
 
-## Check whether source code has been formatted:
-if ! ./scripts/format-code --quiet --whatif; then
-    echo "code not formatted correctly, IGNORING FOR NOW!"
-    #exit_ "CI failed: please run ./scripts/format-code"
+## Check whether C/C++ source code has been formatted:
```

Comment:
Good idea to keep it a warning for now. Am I right that `format-c-code` checks/formats only the files in a (staged?) commit?

Reply:
This PR doesn't enable C code formatting. I haven't looked at the details of that yet.

Reply:
Sure, sounds good. It just looked like you're calling the C formatting script here.